.github/workflows/build.yml

```
@@ -32,17 +38,39 @@ jobs:
         sudo make install
         sudo ldconfig
         
+    - name: Debug matrix values
+      run: |
+        echo "matrix.noscrypt=${{ matrix.noscrypt }}"
+        echo "matrix.noscrypt_name=${{ matrix.noscrypt_name }}"
+        echo "Condition result: ${{ matrix.noscrypt == 'ON' }}"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)